docs/transport/transport.md

```diff
@@ -0,0 +1,65 @@
+# gittuf Remote Transfer Protocol
+
+Last Modified: July 1, 2024
+
+## Introduction
+
+Alongside the `gittuf` binary, gittuf ships with a custom remote transfer
```

Comment:
let's link to and highlight right off the bat that it implements the git remote-helper interface